.github/workflows/release.yml

```
@@ -65,123 +68,127 @@
+  build:
     runs-on: ubuntu-latest
     outputs:
-      dist_path: dist
+      dist_dir: dist
     steps:
       - uses: actions/checkout@v4
         with:
           fetch-depth: 0
-          # push → new tag; PR → HEAD
+          # for push: checkout the tag we just made
           ref: ${{ github.event_name == 'push' && needs.tag-version.outputs.new_tag || '' }}
```

actionlint (1.7.7)

property "tag-version" is not defined in object type {}

(expression)